src/scene_battle_rpg2k3.cpp

```diff
@@ -321,6 +321,46 @@ void Scene_Battle_Rpg2k3::CreateUi() {
 		SetupSystem2Graphics();
 	}
 
+	if (lcf::Data::battlecommands.window_size == lcf::rpg::BattleCommands::WindowSize_small) {
+		int height = 68;
+		int y = SCREEN_TARGET_HEIGHT - height;
+
```

Comment:
Except for ``status_window`` this looks the same for everything.

Did not launch a compiler but something like this should work:

```cpp
for (auto* window: { options_window, command_window, skill_window, item_window, target_window }) {
  if (window) {
     // Height, Y, BorderY, SetContents, SetMenuItemHeight
  }
}
```

Reply:
static typing is in the way here, lets use a lambda instead:

```cpp
auto small_window  = [&](auto* window) {
  if (window) {
     // Height, Y, BorderY, SetContents, SetMenuItemHeight
  }
};

small_window(options_window);
small_window(command_window);
...
```

Reply:
Updated.